views/partials/head.html

```diff
@@ -80,7 +82,7 @@
       {% include "/assets/css/case-study.min.css" %}
     {% elif doc.view == '/views/list_page.html' or doc.url.path.endswith('/latest/') %}
       {% include "/assets/css/list.min.css" %}
-    {% elif doc.view == '/views/blog_detail.html' or doc.url.path.find('/roadmap/') != -1 %}
+    {% elif doc.view == '/views/blog_detail.html' or doc.url.path.endswith('/roadmap/') %}
```

Comment:
This one needs to be reverted, as the `find` syntax is needed to not break the roadmap filter pages.

Reply:
Thanks for catching that. It's an artifact of my attempt to merge my fork to reflect changes on master :-/